docs/choreolib/auto-routines.md

```diff
@@ -166,7 +164,7 @@ public AutoRoutine fivePieceAutoTriggerSeg(AutoFactory factory) {
 ### Creating an auto routine with triggers and a monolithic trajectory
 
 ```java
-public AutoRoutine fivePieceAutoTriggerMono(AutoFactory factory) {
```

Comment:
not sure if this was the mistake or what you returned was a mistake, but it didn't match before

Reply:
bah why doesn't it show the full diff, i swapped AutoRoutine with Command because that's what it actually returns.